Assets/MixedRealityToolkit.Examples/Demos/EyeTracking/General/Profiles/EyeTrackingDemoSceneSystemProfile.asset

```diff
@@ -0,0 +1,72 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!114 &11400000
+MonoBehaviour:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_GameObject: {fileID: 0}
+  m_Enabled: 1
+  m_EditorHideFlags: 0
+  m_GeneratorAsset: {fileID: 0}
+  m_Script: {fileID: 11500000, guid: b81dae1d46879b444aa9847f7961649f, type: 3}
+  m_Name: EyeTrackingDemoSceneSystemProfile
+  m_EditorClassIdentifier: 
+  isCustomProfile: 1
+  useManagerScene: 1
+  managerScene:
+    Name: EyeTrackingDemo-00-RootScene
+    Path: Assets/MixedRealityToolkit.Examples/Demos/EyeTracking/Scenes/EyeTrackingDemo-00-RootScene.unity
+    Included: 1
+    BuildIndex: 0
+    Tag: Untagged
+    Asset: {fileID: 3671201678493226958, guid: 27c3fbe0128089f48994cdc6af8668fc, type: 3}
```

Comment:
When first trying out this branch, the 'Scene System' showed me that the linked scenes were missing. After some testing, I found out that this was due to the fileID. When I replaced '3671201678493226958' with '102900000' for all entries, without changing anything else, it would correctly detect all the scenes in the Editor again.

Reply:
Looks like this is a [Unity bug.](https://issuetracker.unity3d.com/issues/sceneasset-references-go-missing-when-upgrading-a-project-from-2018-dot-3-and-earlier-to-2019-dot-1-or-2019-dot-2) I've updated the scene info drawer to automatically by path if the asset is missing. Hopefully this won't be necessary after we all move to 2019.